source/moon-resolution.js

```diff
@@ -69,6 +71,10 @@
 			type = type || screenType;
 			if (type) {
 				enyo.dom.addBodyClass('moon-res-' + type.toLowerCase());
+				var scrObj = getScreenTypeObject(type);
+				if (scrObj.aspectRatioName) {
+					enyo.dom.addBodyClass('enyo-aspect-ratio-' + scrObj.aspectRatioName.toLowerCase());
```

Comment:
Should we consider adding enyo-res above (in addition to the moon-res)? That way devs who wanted to could move to the full future-proof version right away.